pkgs/tools/misc/radeon-profile/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, qt5, xorg }:
+
+stdenv.mkDerivation rec {
+
+  name = "radeon-profile-${version}";
+  version = "20161221";
+
+  nativeBuildInputs = [ qt5.qmakeHook ];
+  buildInputs = [
+    qt5.qtbase
+    xorg.libXrandr
+  ];
+
+  src = (fetchFromGitHub {
+    owner  = "marazmista";
+    repo   = "radeon-profile";
+    rev    = version;
+    sha256 = "0zdmpc0rx6i0y32dcbz02whp95hpbmmbkmcp39f00byvjm5cprgg";
+  }) + "/radeon-profile";
+
+  installPhase = ''
```

Comment:
To override a phase like this, the first line must be `runHook preInstall` and the last line must be `runHook postInstall`.